docs/3.0/basic-filters.md

```diff
@@ -90,7 +90,7 @@ end
 
 ## Filter types
 
-Avo has several types of filters available [Boolean filter](#boolean_filter), [Select filter](#select_filter), [Multiple select filter](#multiple_select_filter) and [Text filter](#text_filter).
+Avo has several types of filters available [Boolean filter](#boolean_filter), [Select filter](#select_filter), [Multiple select filter](#multiple_select_filter), [Text filter](#text_filter) and since version <Version version="3.11.7" /> [Date time filter](#date_time_filter).
```

Comment:
```suggestion
Avo has several types of filters available [Boolean filter](#boolean_filter), [Select filter](#select_filter), [Multiple select filter](#multiple_select_filter), [Text filter](#text_filter) and since version <Version version="3.11.8" /> [Date time filter](#date_time_filter).
```